www/src/contexts/QueueContext.ts

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react'
+
+export default createContext < any >({})
```

Comment:
Can we provide interface with types for the context?

Reply:
We would need https://github.com/pluralsh/plural/pull/341 to get merged first since those are the API types. I have manually added some partial types just to simplify the development, but I'd prefer to use generated types here.

Reply:
Ok, sure.